lib/l10n/app_en.arb

```diff
@@ -119,9 +119,17 @@
     "country_dm" : "Dominica",
     "country_do" : "Dominican Republic",
```

Comment:
Do you think maybe we should remove the not-used `country_XX` strings?
There are only 14 country codes in the new table, but we have much more country codes in the `.arb` files.
It's not a problem as of itself, but it will create extra work for translators, while most of the translated by them fields won't be used by the app.

Reply:
i was thinking that too, but some people already translated the strings. so i was thinking removing them might make them have double work when we do re-add them. 
so i'm not sure...

Reply:
It's double work for them when we re-add them, or it's double work for other translators who haven't translated the strings yet :)
I think it would be better to do what would simplify the code